.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,79 @@
+name: Release from tags
+# Create a GitHub release when a tag is pushed.
+
+on:
+  push:
+    tags:
+      # this is a glob, not a regexp
+      - 'anaconda-*'
+
+permissions:
+  contents: write
+
+jobs:
+  release-from-tag:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Extract version from the tag
+        id: get_version
+        # If the version does not fit the regexp, we'll just fail. TODO maybe abort instead?
+        run: |
+          echo "Checking pushed tag ref ${{ github.ref }}"
+          VER=$(echo ${{ github.ref }} | perl -ne '/^refs\/tags\/anaconda-([0-9]+(?:\.[0-9]+){1,3})-1$/ && print "$1\n";')
+          if [ -z "$VER" ] ; then
+            echo "Tag ref ${{ github.ref }} is not a valid release tag."
+            exit 1
+          else
+            echo "Tag ref ${{ github.ref }} detected as release version $VER."
+            echo "::set-output name=version::$VER"
+          fi
+
+      - name: Check out repo
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+
+      #- name: Build anaconda-rpm container (for RPM build)
+      #  run: |
+      #    make -f Makefile.am anaconda-rpm-build
+
+      - name: Run the build in the container
+        run: |
+          mkdir /tmp/results
+          podman run \
+            -i \
+            -v /tmp/results:/results \
+            -v `pwd`:/anaconda \
+            quay.io/rhinstaller/anaconda-rpm:master <<EOF
+
+          # prepare environment
+          set -eux
+
+          # install dependencies for translation canary
+          dnf install -y git python3-polib python3-pip
+          pip install --upgrade pip pocketlint
```

Comment:
Maybe. The tar build runs translation checks, otherwise it's more like rpm build than unit tests. I don't really want to make yet more containers. Jirka doesn't want more dependencies.

I ditched a srpm build that followed the tar build, so it might be useful to re-examine if the ci container is enough.